website/docs/d/monitor_diagnostic_categories.html.markdown

```diff
@@ -32,7 +32,9 @@ data "azurerm_monitor_diagnostic_categories" "example" {
 
 * `id` - The ID of the Resource.
 
-* `logs` - A list of the Log Categories supported for this Resource.
```

Comment:
we can't just remove logs, it will need to be depreciated in favour of the new property (and flagged correctly behind the 4.0 flag)

Reply:
What if we leave 'logs', and just add 'logs_category' in this release? It would be consistent with current naming convention.

Reply:
Hi @katbyte, if i may ask: i see now the "logs" has been deprecated and instead comes log_category_types and log_category_groups two new ones... it is rather confused, which one of them is the one we should use to replace "logs"? Thank you very much.